getting_started/bootstrap.rst

```diff
@@ -77,11 +97,35 @@ Start by creating a ``terraform.tfvars`` file. There is a template that you can
 - **edge_count**: number of egde nodes to be created
 - **edge_flavor**: an instance flavor for the edge nodes
 
+If you are wondering yet again where you can fetch correct flavor label names then no worries, you are not being a stranger here. The nova command-line interface will come in handy. Just run the following command::
```

Comment:
Same here as in the above long comment. Let me know we still would like to move to troubleshooting.